tests/test_tutorials.py

```diff
@@ -74,6 +77,7 @@ def get_cell_by_name(nb, name):
     raise ValueError("no cell named '{}'".format(name))
 
 
+@pytest.mark.skipif(skip_py2, reason=py2_deprecated)
```

Comment:
should we just add a check in `__init__.py` that raises an error if trying to run from py2? any thoughts on this @khaeru ?

Reply:
This is just a quick fix to avoid the `from __future__ import division` in the tutorial.

I'd suggest to add proper warning messages when we remove py2-tests completely after release 1.2.